src/app/main.rs

```diff
@@ -402,7 +400,11 @@ where
             &config,
             local_identity.clone(),
             outbound_listener.local_addr(),
-            resolver,
+            outbound::resolve(
+                config.destination_get_suffixes.clone(),
+                outbound::ExponentialBackoff::default(),
```

Comment:
nit: is `outbound::resolve` ever called with a value for this that isn't the default? if not, maybe it should just always use default?

Reply:
it should probably be wired through to the config